README.md

```diff
@@ -120,6 +144,38 @@ Attempts to play a given track in the queue.  If this track is not in the queue
 **Returns:**
 - retVal `Promise` - A promise that resolves when the track starts playing.  This promise can be rejected so you need to handle any errors with `.catch`
 
+### Search
+#### `search.getResults()`
```

Comment:
Maybe rename this to `getCurrentResults()` for consistency with `queue.getCurrentSong()`? And to imply there has been prior required action (e.g. adding to queue, performing a search)